modules/data/symbols/patches/language/pokeleafgreen.yml

```diff
@@ -0,0 +1,645 @@
+#-----------------#
+Task_StartMenuHandleInput:
+  D: 0x806f154
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+sMenu:
+  J: ~
+sStartMenuCursorPos:
+  J: ~
+sNumStartMenuItems:
+  J: ~
+sStartMenuOrder:
+  J: ~
+#-----------------#
+
+#--------------#
+#   Bag Menu   #
+#--------------#
+CB2_BAGMENURUN:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+Task_BagMenu_HandleInput:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+Task_FieldItemContextMenuHandleInput:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+gBagMenuState:
+  J: ~
+gPaletteFade:
+  J: ~
+#---------------#
+
+#----------------#
+#   Party Menu   #
+#----------------#
+CB2_UPDATEPARTYMENU:
+  D: 0x811ebac
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+Task_PrintAndWaitForText:
+  D: 0x8120334
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+Task_HandleInput:
+  D: 0x809f328
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+Task_HandleSelectionMenuInput:
+  D: 0x8122c6c
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+Task_SlideSelectedSlotsOnscreen:
+  D: 0x8123398
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+#----------------#
+
+#-------------------------#
+#    PC Menu / Storage    #
+#-------------------------#
+Task_PCMainMenu:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+Task_PokeStorageMain:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+gPokemonStoragePtr:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+#-------------------------#
+
+#--------------#
+#   Listeners  #
+#--------------#
+CB2_EggHatch_0:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+CB2_EggHatch_1:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+EventScript_RepelWoreOff:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+EventScript_DoTrainerBattle:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+EventScript_DoTrainerBattleFromApproach:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+EventScript_FieldPoison:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+#---------------#
+
+# Ununsed symbols that were conflicting with currently used symbols
+# We set them to 0x0 to 'get them out of our way' and correctly target the desired symbol
+PokemonMansion_EventScript_DontPressSwitch:
+  D: 0x0
+Route19_Text_LucRematchIntro:
+  D: 0x0
+Route21_North_EventScript_IanRematch:
+  D: 0x0
+EventScript_CutTree:
+  F: 0x0
+Route23_EventScript_MissingBoulderBadge:
+  F: 0x0
+EventScript_SetEnteringCyclingRoad:
+  F: 0x0
+Route23_EventScript_MissingBadge:
+  F: 0x0
+NavelRock_Summit_EventScript_HideHoOh:
+  F: 0x0
+CableClub_Text_YouMayBattleHere:
+  I: 0x0
+Status_Freeze:
+  I: 0x0
+EventScript_PkmnCenterNurse_PlayerWaitingInUionRoom:
+  I: 0x0
+BattleScript_BrickBreakDoHit:
+  S: 0x0
+BattleScript_BrickBreakAnim:
+  S: 0x0
+EventScript_Open9FDoor3:
+  S: 0x0
+Text_DontBeLikeThat:
+  S: 0x0
+Trade_Text_WhatThatsNoMon:
+  J: 0x0
+
+# Unused
+Task_ExitNonDoor:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+Task_ContinueTaskAfterMessagePrints:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+CB2_BEGINEVOLUTIONSCENE:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+CB2_EVOLUTIONSCENEUPDATE:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+TASK_PLAYCRYWHENRELEASEDFROMBALL:
+  D: ~
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+  
```

Comment:
They are unused by essence, so we don't need them here. I think we can remove them